api.swagger.yaml

```diff
@@ -12263,7 +12256,34 @@ components:
               type: boolean
               example: false
               readOnly: true
+            free_skills:
```

Comment:
free skils is a taxonomy, should be an object array like others. http://invotra.test/api/0.3/free_skills/list